codes/quantum/qubits/stabilizer_over_gf4.yml

```diff
@@ -0,0 +1,27 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: stabilizer_over_gf4
+physical: qubits
+logical: qubits
+
+name: 'Stabilizer code over \(GF(4)\)'
+introduced: '\cite{arXiv:quant-ph/9608006}'
+
+description: 'An \([[n,k,d]]\) stabilizer code whose encoding is based on self-orthogonal \([n, n-k, d^*]\) code of \(GF(4)^n\) with respect to trace inner product where \(d \ge d^*\).'
+
+protection: 'Detects errors on \(d-1\) qubits, corrects errors on \(\left\lfloor (d-1)/2 \right\rfloor\) qubits.'
+
+notes:
+  - '\(GF(4)\) constists of \(\{0, 1, w, \bar{w}\}\), with \(\bar{w} = w^2 = w + 1\), \(\mathrm{Tr}(x) = x+\bar{x}\) and trace inner product \(u * v = \mathrm{Tr}(u \cdot \bar{v})\).'
+
+relations:
```

Comment:
Why did you make css as cousin? I would remove it unless you had something in mind. Also, make q-ary_linear a cousin and justify by saying that q=4 q-ary linear code is used in this construction.